extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -142,14 +144,25 @@ require([
 	highImpact,
 	inContent,
 	doc,
+	instantGlobals,
+	tracker,
 	trackingOptIn,
 	win
 ) {
 	'use strict';
 
 	function initDesktopSlots() {
 		highImpact.init();
-		inContent.init('INCONTENT_PLAYER');
+		if (instantGlobals.wgDisableIncontentPlayer) {
```

Comment:
This is not an instant global. We need to pass this variable in the `AdEngine2ContextService`